features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/site/public/locales/en.json

```diff
@@ -230,33 +230,33 @@
   "KeyManagers.AddEditKeyManager.form.Issuer": "Issuer",
   "KeyManagers.AddEditKeyManager.form.add": "Add",
   "KeyManagers.AddEditKeyManager.form.authorizeEndpoint": "Authorize Endpoint",
-  "KeyManagers.AddEditKeyManager.form.authorizeEndpoint.help": "Ex: https://localhost:9443/oauth2/userinfo",
+  "KeyManagers.AddEditKeyManager.form.authorizeEndpoint.help": "E.g., https://localhost:9443/oauth2/userinfo",
```

Comment:
isn't it common to use `Eg.` instead of  `E.g.`?

Reply:
The correct way is "e.g.,". And that is what we have used in the documentation as well.

Reply:
ok, thanks for the clarification.